core/database/src/main/kotlin/app/pachli/core/database/AppDatabase.kt

```diff
@@ -317,6 +317,67 @@ val MIGRATE_8_9 = object : Migration(8, 9) {
     }
 }
 
+/**
+ * Clears references to deleted accounts.
+ *
+ * The migration from 10 -> 11 adds FK relationships to the AccountEntity table. Because of
+ * earlier bugs the child tables in those relationships may contain orphoned rows that
```

Comment:
```suggestion
 * earlier bugs the child tables in those relationships may contain orphaned rows that
```

Reply:
> ```suggestion
>  * earlier bugs the child tables in those relationships may contain orphaned rows that
> ```

652203257afd2a244e9a816992022222e69bd86c